forks/Microsoft.Health.FellowOakDicom/Microsoft.Health.FellowOakDicom.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>library</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>disable</Nullable>
+    <AnalysisLevel></AnalysisLevel>
```

Comment:
What happens if you leave out `AnalysisLevel`, `AnalysisMode`, and `EnforceCodeStyleInBuild`? Shouldn't the rules specified in the .editorconfig/.globalconfig account for these?

Reply:
Removed the AnalysisMode. But AnalysisLevel and EnforceCodeStyleInBuild are being used; removing them will cause the code compilation failures (CLSCompliant and code style errors).

Reply:
Are there too many to include in project-specific global config?

Reply:
I added the globalconfig for "Forks". So that in-case if we fork something else, as long as that code lives inside "forks" this globalconfig would apply for that too.